transcription/tasks.py

```diff
@@ -0,0 +1,145 @@
+"""
+Celery tasks for transcription processing.
+"""
+
+import logging
+from celery import shared_task
+from django.core.exceptions import ObjectDoesNotExist
+
+from .services import TranscriptionService, TranscriptionConfig
+from .models import SessionAudio, AudioTranscript
+
+logger = logging.getLogger(__name__)
+
+
+@shared_task(bind=True, max_retries=3, default_retry_delay=60)
```

Comment:
Since this is calling the whisper api, should I be concerned about retries? Is that likely to result in the same audio partially transcribed multiple times, resulting in an unexpected bill? Or is that not a worry here?

Also please tell me about the retry_delay. Does it retry if the task hasn't finished, or just if it hasn't started or something?